x/programs/rust/wasmlanche-sdk/src/state.rs

```diff
@@ -14,6 +14,22 @@ pub enum Error {
     Deserialization,
 }
 
+/// Gets the balance for the specified address
+/// # Panics
+/// Panics if there was an issue deserializing the balance
+#[must_use]
+pub fn get_balance(account: Address) -> u64 {
```

Comment:
The `get_balance` seems to rather belong to the `program.rs` file, is doesn't really deal with state.

Reply:
It wouldn't deal with `program` either in the case of an EOA, so it is left here.